.nvmrc

```diff
@@ -1 +1 @@
-20.9.0
\ No newline at end of file
+20.10.0
```

Comment:
the reassure tests seem to be failing because of this, why did you update it here?

Reply:
yes, because @roryabraham asked me to do so: https://github.com/Expensify/App/issues/12775#issuecomment-1899331180

Reply:
Thanks!